public/r2/v1/swagger/swagger.json

```diff
@@ -442,7 +464,7 @@
                     "200": {
                         "description": "Ruleset updated",
                         "schema": {
-                            "$ref": "#/definitions/ApiResponse"
+                            "$ref": "#/definitions/DeleteResponse"
```

Comment:
A little confused about the DeleteResponse here - why do we need to return the version for this?

Reply:
well, the version is for a rule set and this operation updates the current ruleset snapshot, there will be one less rule. If you then wanted to do further modifications after the delete you would be operating on a stale version of the rule set so it needs to be returned.